src/configuration/app/variables.md

```diff
@@ -0,0 +1,115 @@
+# Variables
+
+Platform.sh provides a number of ways to set [environment variables](/development/variables.md), either globally or specific to a single environment.  For values that should be consistent between different environments (because they're configuring the application or runtime itself, generally) the easiest way to control them is to set them in the `.platform.app.yaml` file.
+
+Only prefixed variables may be set from the `.platform.app.yaml` file.  Some prefixes have specific meaning while others are only significant to a particular application.  Nested variables will be automatically converted into a nested array or list structure as appropriate to the language.
+
+For example, the following section in `.platform.app.yaml` will set a single variable named `env:AUTHOR` to the value `Larry`.
+
+```yaml
+variables:
+    env:
+        AUTHOR: 'Larry'
+```
+
+That will have the exact same runtime effect as setting the variable via the CLI as follows, except it will be versioned along with the code:
+
+```bash
+$ platform variable:set env:AUTHOR Larry
+```
```

Comment:
Isn't this more similar to `project:variable:set`, in effect? As in, you can use it at build time as well as runtime